src/oas.md

```diff
@@ -1192,17 +1192,17 @@ Using `content` with a `text/plain` media type is RECOMMENDED for `in: "cookie"`
 
 ##### Style Values
 
-In order to support common ways of serializing simple parameters, a set of `style` values are defined.
+In order to support common ways of serializing simple parameters, a set of `style` values are defined. Combinations not represented in this table are not permitted.
```

Comment:
That adds a MUST NOT requirement, which we cannot do in a patch release.

Reply:
This requirement already existed; it is not adding a new one. It was just not mentioned here.

Reply:
It's the table itself here that sets out the permitted combinations of `style`s and `in`. But this is the way it's been since at least 3.0, and the schemas also set out these restrictions (see the "styles-for-path", "styles-for-header" etc definitions) -- so I believe the very strong intent is that no other combinations are permitted.